api/db-cdr/generate-cdr/README.md

```diff
@@ -0,0 +1,37 @@
+COUNT GENERATION
+
+This document describes what counts are generated for each of the data domains
+
+Total Person count is generated and stored with analysis id 1.
+Person count by gender is generated and stored with analysis id 2.
+Person count by year of birth is generated and stored with analysis id 3.
+Person count by race is generated and stored with analysis id 4.
+Person count by ethnicity is generated and stored with analysis id 5.
+Person count by year of birth and gender is generated and stored with analysis id id 10.
+Person count by race and ethnicity is generated and stored with analysis id 12.
+
+
+1. EHR DATA
+
+   Domains( VISIT, CONDITION, PROCEDURE, DRUG, OBSERVATION, MEASUREMENT):
+    a: Counts and source counts of people with one of the occurrence of the concepts in these domains are generated and stored in analysis id 3000.
```

Comment:
Clear wording "Counts and source counts of distinct patients with at least one occurrence ..."